CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Add option to GriddedIO class so that if an unitilized time object is passed, the resulting file will not depend on time
+
```

Comment:
I did not see any changes to interfaces, so I suspect that this is not an "option" per se, but rather a change to the logic to improve treatment of uninitialized time objects.    Please clarify the text.

Reply:
@tclune, yes small change in logic to detect an uninitialized time object. That way @metdyn for his swath work, can just pass an uninitialized time object into the class, and that's fine, it will not be used, and griddedIO will produce files where there is no time dimension and the variables do not depend on time.

Reply:
@bena-nasa  Thanks for the un_initialized time option! I am not there yet, but look forward to use this griddedIO and this functionality.

Reply:
@bena-nasa  I think you should merge this PR. I will test it next week. Thanks!

Reply:
I tested this code by merging to my local branch and find no effect on existing output.